src/Microsoft.Performance.SDK/Processing/OptionInstance.cs

```diff
@@ -77,5 +77,25 @@ public OptionInstance(object optionId, IEnumerable<string> arguments)
         ///     Gets the arguments provided to this option.
         /// </summary>
         public IEnumerable<string> Arguments { get; }
+
+        public override bool Equals(object obj)
```

Comment:
Do we need to override this and GetHashCode?

Reply:
Yep! To be able to compare OptionInstances via ProcessorOptions for testing.